tests/candev/Makefile

```diff
@@ -9,11 +9,11 @@ USEMODULE += isrpipe
 CAN_DRIVER ?= MCP2515
 
```

Comment:
Can't we 

    USEPKG += libsocketcan

to make sure this is compile tested by CI?

Reply:
By adding the following in a `Makefile.board.dep`, that should be possible:

```mk
ifeq (native,$(BOARD))
  USEPKG += libsocketcan
endif
```

Reply:
Or just in the `Makefile.dep` of native cpu if the periph_can feature is used => that would be my preferred solution